### PR TITLE
feat: update nemeton results page

### PR DESCRIPTION
**Pull request by houssemmimoun27**
This PR update the nemeton builders program results page.

### CODE REVIEW

**Review by antho31 — APPROVED** [review]

**Review by ccamel — APPROVED** [review]
LGTM!

File: src/data/builder/builder.ts [hunk 1]
```diff
@@ -9,9 +9,13 @@ export const buildersData: BuildersDTO = [
       'OKP4 Governance Creator',
       'OKP4 Law-Stone Builder',
       'OKP4 Objectarium Explorer',
-      'OKP4 Young Druid'
+      'OKP4 Young Druid',
+      'OKP4 Dataverse precursor',
+      'OKP4 Prolog query master',
+      'OKP4 interface innovator',
+      'OKP4 visionary contributor'
```
Comment (fredericvilcot) [inline src/data/builder/builder.ts:16]: @antho31 
The terms above (lines 8-12)  are capitalized, wouldn't it be better to keep this formatting?

Comment (antho31) [reply src/data/builder/builder.ts:16]: Yes you're right, better capitalized

Comment (antho31) [reply src/data/builder/builder.ts:16]: @houssemmimoun27 can you fix it? :)

**Review by fredericvilcot — APPROVED** [review]

**Pull request merged**